app/docker/networks/queries.ts

```diff
@@ -0,0 +1,45 @@
+import { useQuery, useMutation, useQueryClient } from 'react-query';
+
+import { EnvironmentId } from '@/portainer/environments/types';
+import {
+  error as notifyError,
+  success as notifySuccess,
+} from '@/portainer/services/notifications';
+
+import { getNetwork, removeNetwork } from './network.service';
+import { NetworkId } from './types';
+
+export function useNetwork(environmentId: EnvironmentId, networkId: NetworkId) {
+  return useQuery(
+    ['environmentId', environmentId, 'docker', 'network', networkId],
+    () => getNetwork(environmentId, networkId),
+    {
+      onError: (err) => {
+        // if there's an error fetching the network, tell the user
+        notifyError('Failure', err as Error, 'Unable to get network');
+      },
+    }
+  );
+}
+
+export function UseDeleteNetwork(
```

Comment:
the only place where we use PascalCase (upper cased camelCase) is for components and classes.

```suggestion
export function useDeleteNetwork(
```